author/job.py

```diff
@@ -260,7 +261,17 @@ def submit(self, dump_job=True, expandchunk=False, **kwargs):
 
             return ""
 
-        job_id = self.spool(owner=getpass.getuser())
+        # allow to override the host and port configured via TRACTOR_ENGINE string
+        hostname, port = None, None
+        if TRACTOR_ENGINE:
+            _ = TRACTOR_ENGINE.split(":")
+            hostname, port = _[0], int(_[1])
+
+        job_id = self.spool(
+            owner=getpass.getuser(),
+            hostname=kwargs.get("hostname", hostname),
+            port=kwargs.get("port", port)
```

Comment:
We should first check whether both the hostname AND the port have been passed to the function, otherwise we'll get the name from one source and the port from another.
If we want to use a default port we should split the config into `TRACTOR_ENGINE_HOST` and `TRACTOR_ENGINE_PORT` so they can easily be configured separately.